api/config/v1alpha1/envoygateway_types.go

```diff
@@ -86,6 +95,47 @@ type FileProvider struct {
 	// TODO: Add config as use cases are better understood.
 }
 
+// RateLimit defines the configuration associated with the Rate Limit Service
+// used for Global Rate Limiting.
+type RateLimit struct {
+	// Backend holds the configuration associated with the
+	// database backend used by the rate limit service to store
+	// state associated with global ratelimiting.
+	Backend *RateLimitDatabaseBackend `json:"backend"`
```

Comment:
This should not be required since it's required. Xref the [Kube API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md) if needed.

Reply:
thanks for sharing the link, fixed